Core/NoOpRunner.Core/Shapes/EntityShapes/DamageCrystal.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace NoOpRunner.Core.Shapes.EntityShapes
 {
-    class DamageCrystal : EntityShape
+    public class DamageCrystal : EntityShape
```

Comment:
You could revert these now that you have the `InternalsVisibleTo` Attribute

Reply:
Fixed